docs/manage/get_support.rst

```diff
@@ -56,16 +56,38 @@ The ``sqlite3`` command is necessary. This can be installed with ``sudo apt inst
       sqlite3 db.sqlite3 .dump | sqlite3 fixed.db
       mv fixed.db db.sqlite3
 
-#. Remove the following files.
+#. Remove temporary database files.
    
     .. code-block:: bash
 
-      rm -f db.sqlite3-wal db.sqlite3-shm job_storage.sqlite3 /process_cache/cache.db
+      rm -f db.sqlite3-* job_storage.sqlite3* notifications.sqlite3* process_cache/cache.db
 
 #. Restart Kolibri.
 
 For further assistance, please report the issue on our `Community Forums <https://community.learningequality.org/>`_, stating the operating system and Kolibri version.
 
+.. tip::
+
+  The above instructions work on *malformed* databases, but **if your SQLite3 version is 3.29 or newer**, you can use the `.recover command <https://sqlite.org/cli.html#recover>`__ to restore other types of database corruption as well. This can happen if Kolibri is terminated abruptly or if your storage media fails.
+
+  You can see your SQLite3 version by running ``sqlite3 --version`` from command line. If your database is corrupted but your SQLite3 is too old to run ``.recover``, consider copying your ``db.sqlite3`` file to a different system with a new SQLite3.
+
+  To apply ``.recover`` on a broken database, open up your Terminal and run a set of commands similar to when we applied ``.dump``:
+
+  .. code-block:: bash
```

Comment:
Could you rework this code block to be separated commands instead (like I did with the first set of DB repair commands few weeks ago)? Remember that anything inside the `code-block` directive does not get localized, which leaves all the instructions in the comments not translated. 

I had the full intention to use the _Suggest_ feature, but it just becomes too messy to do in this editor, plus given that all is inside the `tip` block, I have no way of checking if the proper formatting is preserved  :confused:

Reply:
I found and tested a settting that marks both image alt texts and literal blocks for translation. This means that the code example now shows up for translation.

This adds a new challenge for literal blocks that previously worked in all languages: Getting translations to 100% means to generate zero-modification copies in Crowdin.

@radinamatic is it of any concern that these blocks are generally marked translatable? Won't translators for instance already know to not make mistakes in programmatic syntax and be sure to use copy-paste rather than typing manually?

![image](https://user-images.githubusercontent.com/374612/78555318-0ca0e380-780d-11ea-94b5-57ab732563b4.png)

Reply:
Kudos to you for noticing that we weren't extracting the alternative text for images and translating them, that will be a great improvement and a welcome addition. :+1: :100: 

However, I'm not a fan at all of using this approach for literal code blocks. It makes the job unnecessarily more complicated for translators who have to **intermittently translate and leave unmodified the lines** in one same _presumably_ translatable segment...? 

And all for what exactly, just so we could save some text length by not repeating the ``.. code-block::`` directive? You yourself admitted that copy-pasting long lists of commands is a bit of a mal-practice. I would argue that clearly ordered steps with single line of commands in each have higher readability and promote better understanding of the code one is executing (and eventual errors that might be raised), compared to copy-pasting entire chunks.

Reply:
Sounds good -- for this one-off case, I will adapt it to the chunked version to not mix commands into translatable contents.

It frees up screen space, makes it easier to write documentation and translate it as well. It also makes it better for the reader, as they get an option to copy-paste the entire instruction: Note that the above instruction actually was written for that exact purpose. That also makes it easier to test the instructions :)

Yes, it means that the translator has to know that they are translating syntax, and I think this might ultimately be a sort of soft requirement.. so either translators already know this, or they become familiar with it while translating. If not, we might implement automated steps or contextual instructions (you know those in-context messages for translators).